### ISSUE TITLE
Table view for "Diagnosics"

### ISSUE THREAD

**Issue by sloniki** [ElasticHQ/elasticsearch-HQ#404]
Please please return table view for Node Diagnostics!
Old version had more convenient view.

**Comment by royrusso** [comment]
Isn't this the same layout? Or do you mean just one large table... so column per node?

**Comment by sloniki** [comment]
Yes, one big table with column per node is more readable, than separate panels

**Comment by royrusso** [comment]
Ok. Worth consideration for easy comparison. Downside is too many nodes cause side-ways scroll.
